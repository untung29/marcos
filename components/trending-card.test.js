import React from "react";
import renderer from "react-test-renderer";
// import { shallow } from "enzyme";
import TrendingCard from "./trending-card";
import { withNavigation } from "react-navigation";

it("<TrendingCard />", () => {
  const tree = renderer.create(<TrendingCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
