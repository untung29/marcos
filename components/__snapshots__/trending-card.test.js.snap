// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrendingCard /> 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2F5487",
      "borderRadius": 8,
      "flexDirection": "row",
      "justifyContent": "space-around",
      "padding": 16,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "height": 24,
        "width": 24,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "minWidth": 128,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontFamily": "PoppinsRegular",
          },
          undefined,
        ]
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontFamily": "PoppinsRegular",
          },
          undefined,
        ]
      }
    >
      Volume:
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontFamily": "PoppinsBold",
          },
          Object {
            "marginTop": 16,
          },
        ]
      }
    >
      $
    </Text>
  </View>
</View>
`;
